src/app/posts/post-detail/post-detail.component.css

```diff
@@ -35,12 +35,7 @@
 
 .post-details {
   width: 70%;
-  margin: 0 auto;
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translateX(-50%);
-
+  margin: 20px auto 0 auto;
```

Comment:
simply: 20px auto 0 ; // the  "auto" in the center is for  left/right margins (this is a just a shorterhand syntax)